crates/namada/src/vm/host_env.rs

```diff
@@ -2003,17 +1991,20 @@ where
         max_signatures,
         || gas_meter.borrow_mut().consume(gas::VERIFY_TX_SIG_GAS),
     ) {
-        Ok(_) => Ok(HostEnvResult::Success.to_i64()),
+        Ok(_) => Ok(()),
         Err(err) => match err {
             namada_tx::VerifySigError::Gas(inner) => {
-                sentinel.borrow_mut().set_out_of_gas();
                 Err(vp_host_fns::RuntimeError::OutOfGas(inner))
             }
-            namada_tx::VerifySigError::InvalidSectionSignature(_) => {
-                sentinel.borrow_mut().set_invalid_signature();
-                Ok(HostEnvResult::Fail.to_i64())
+            namada_tx::VerifySigError::InvalidWrapperSignature => {
```

Comment:
I believe this case can never occur since the wrapper signature is only verified in protocol, right?